modules/event-exporter/variables.tf

```diff
@@ -0,0 +1,23 @@
+variable "chart_version" {
+  type        = string
+  default     = ""
```

Comment:
please provide some chart version by default, to not have situation when the chart gets updated by itself to latest version